Week2/homework/js-exercises/theReadingList.js

```diff
@@ -0,0 +1,28 @@
+'use script'; 
+{
+    let book1 = ["Olaganustu bir gece", "Stefan Zweig", false];
+    let book2 = ["Kuyucakli Yusuf", "Sabahattin Ali", true];
+    let book3 = ["Satranc", "Stefan Zweig", true];
```

Comment:
It might make sense to write it this way (an array of objects), so you can use a loop to go over them:

```
let books = [
  {
    title: "Olaganustu bir gece",
    author: "Stefan Zweig",
    isRead: false
  },
  {
    title: "Kuyucakli Yusuf",
    author: "Sabahattin Ali",
    isRead: true
  },
  {
    title: "Satranc",
    author: "Stefan Zweig",
    isRead: true
  }
];
```